docs/get_started.rst

```diff
@@ -1,17 +1,388 @@
+********************
+
+#. Go to the `Python downloads page <https://www.python.org/downloads/release/python-383/>`_.
+#. Download the latest Python 3 release. Most users should select the "Download Windows x86-64 executable installer" link for the most recent stable release, but it may depend on your particular computer setup.
+#. Use the installation wizard to install:
+
+   * Check the "Add Python <version> to PATH" checkbox.
+   * Click "Install Now".
+
+.. image:: images/windows_python.png
+
+Install via ``pipx``
+********************
+
+``pipx`` ensures that CumulusCI and its dependencies are installed into their own Python environment separate from other Python software on your computer. **We cannot recommend it enough!**
+
+Open your preferred terminal application (e.g. `CMD.exe <https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/cmd>`_ on Windows).
+If you already have your terminal open, close it and reopen it. Enter the following command::
+
+    $ python -m pip install --user pipx
+
+.. image:: images/pipx.png
+
+To permanently modify the default environment variables:
+
+1. Click Start and search for "edit environment variables", or open System properties and click Advanced system settings.
+2. Click the Environment Variables button.
+3. To change System variables, you need non-restricted access to your machine (i.e. Administrator rights). Add the following paths to your PATH environment variable:
+
+   * ``%USERPROFILE%\.local\bin``
+   * ``%USERPROFILE%\AppData\Roaming\Python\Python38\Scripts``
+
+.. image:: images/env-var2.png
+
+.. note::  Be precise when entering these paths. Add them at the very end of the Variable Value already in place. Separate each path by a semicolon (;) with no space between path names.
+
+Open a new command prompt and verify that ``pipx`` is available::
+
+    pipx --version
+
+You should see a version number after entering this command, such as: ``0.12.3.1``.
+
+If you get an error instead, such as ``'pipx' is not recognized as an internal or external command,
+operable program or batch file.``, confirm that your environment variables have been updated.
+
+Finally, install CumulusCI:
+
+Inside your terminal application, enter the following command::
+
+    pipx install cumulusci
+
+Once finished `verify your installation`_.
+
+
+
+Verify Your Installation
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+In a new terminal window, verify that CumulusCI installed correctly by running ``cci version``.
+
+.. code-block:: console
+
+    $ cci version
+    CumulusCI version: 3.20.0 (/path/to/bin/cci)
+    Python version: 3.8.5 (/path/to/bin/python)
+
+    You have the latest version of CumulusCI.
+
+You can also use this command to check whether your CumulusCI installation is up to date.
+
+Still need help? Feel free to submit a question on our `Trailblazer community group <https://trailblazers.salesforce.com/_ui/core/chatter/groups/GroupProfilePage?g=0F9300000009M9ZCAU>`_.
+
+
+
+Setup SFDX 
```

Comment:
```suggestion
Set Up SFDX 
```
Two words when used here as a verb. https://confluence.internal.salesforce.com/display/DOCTEAM/setup%2C+set+up

Reply:
Yeesh, can't believe I missed this one. Egregious.